client/src/pages/signUp/signUp.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './signUp.css';   {/* css for this file */}
+
+function SignUp() {
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class Welcome {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
Ignore